index.html

```diff
@@ -2,89 +2,90 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My Basic Site
+  <title>Will Pacetti</title>
 
-  Leave these links here...
-  But this should be in a comment...
+  <!--Leave these links here...
+  But this should be in a comment...-->
   <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles/style.css">
 </head>
 <body>
-  <div class="bender">
-    <nav class="bender__nav">
-      <div class="box">
-        <div class="abc">
-          <h1 class="abc__first">Shia LaBeouf</h1>
-
-          <div class="abc__second">
-            <ul class="fry">
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#welcome">Welcome</a>
-
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#about">About</a>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="top-nav">
+          <h1 class="top-nav__left">Shia LaBeouf</h1>
+
+          <div class="app__content">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#welcome">Welcome</a>
+
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#about">About</a>
               </li>
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#catchphrases">Quotes</a>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#catchphrases">Quotes</a>
               </li
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#photos">Photos</a>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#photos">Photos</a>
               </li>
             </ul>
 
-        <div>
+          </div>
```

Comment:
Looking at indentation it looks like there might still be one more missing closing div tag?